benchmark_v2/README.md

```diff
@@ -31,6 +31,9 @@ python run_benchmarks.py --upload-to-hf username/benchmark-results
 
 # Upload with a custom run ID for easy identification
 python run_benchmarks.py --upload-to-hf username/benchmark-results --run-id experiment_v1
+
```

Comment:
would it be possible to change to `--push-to-hub` ?

Reply:
Oooof, it's already called that, but forgot to update the readme :)

Reply:
I mean the argument you add. Instead of `--upload-to-hub`, it might be better to change the name to `--push_to_hub` as this is used everywhere in `transformers`.